tools/codediff/diff_report.py

```diff
@@ -634,14 +637,14 @@ class TestDiff:
 
 
 def sanitize_ptx_lines(lines: list[str]) -> list[str]:
-    """Remove comments and translate kernel38 to kernelN"""
+    """Remove comments and remove kernel id"""
     sanitary_lines = []
     for l in lines:
         # Replace mangled kernel names like
-        #   _ZN11CudaCodeGen10kernel1271ENS_6TensorIfLi2ELi2EEENS0_IfLi3ELi3EEES2_
+        #   _ZN76_GLOBAL__N__00000000_37___tmp_kernel_pointwise_f0_c1_r0_g0_cu_8995cef2_3255329nvfuser_pointwise_f0_c1_r0_g0ENS_6TensorIfLi2ELi2EEES1_S1_
         # with
-        #   _ZN11CudaCodeGen7kernelNENS_6TensorIfLi2ELi2EEENS0_IfLi3ELi3EEES2_
-        l = re.sub(r"CudaCodeGen\d+kernel\d+ENS", "CudaCodeGen7kernelNENS", l)
+        #   _ZN76_GLOBAL__N__00000000_37___tmp_kernel_pointwise_cu_8995cef2_3255329nvfuser_pointwiseENS_6TensorIfLi2ELi2EEES1_S1_
+        l = re.sub(r"_f\d+_c\d+_r\d+_g\d+", "", l)
```

Comment:
what about the `___tmp_kernel_pointwise_cu_8995cef2_32553` part (29 is length of the kernel name)? Is this automatically prepended based on the filename? Just to be safe maybe we could use `l = re.sub(r"_tmp_kernel_[a-z0-9_]+nvfuser_[a-z]+_f\d+_c\d*_r\d+_g\d+", "kernel", l)`.